nbs/core.ipynb

```diff
@@ -1762,7 +1784,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
I have the action_files/clean_nbs script to avoid exactly this diff.

Reply:
fixed